doc/GRADERS.md

```diff
@@ -4,6 +4,15 @@ A+ grader service protocols
 The philosophy of A+ is to integrate external grader services into the course environment.
 The graders should be stateless and need only to worry about grading a submission.
 
+## Authentication and Authorization
+
+Every incoming and outgoing request must either have a A+-provided grading
+token as a GET parameter (included in the `submission_url` below) or a JWT
```

Comment:
The A+ protocol gives the token as part of the submission_url. This would require changing the protocol. Other than that, the code change is trivial. But, if we do this change, we probably should just require the use of the JWT token instead,

Reply:
Yes indeed, it wasn't first clear to me that the GET query parameters refer to the old grader protocol whereas the HTTP Authorization header refers to the new JWT tokens.